camel-core/src/main/java/org/apache/camel/impl/verifier/DefaultComponentVerifier.java

```diff
@@ -123,7 +123,7 @@ protected void verifyParametersAgainstCatalog(ResultBuilder builder, Map<String,
             stream(result.getInvalidEnum())
                 .map(entry ->
                     ResultErrorBuilder.withIllegalOption(entry.getKey(), entry.getValue())
-                        .detail(ResultErrorBuilder.attribute("enum.values"), result.getEnumChoices(entry.getKey()))
+                        .detail("enum.values", result.getEnumChoices(entry.getKey()))
```

Comment:
I wonder whether we should use an upper case constant here, too. (and underscore instead of point).

Alternative I we could tune the enums to reflect nicer names. E.g. to lower, _ -> ., adding  a prefix for all standard enums.

However in this case we then can't use name() in the interface but would have to use toString() ( name() is not overwritable, but toString()).

Still we have the benefit to compare via '==' (to check whether its a specific enum) and 'instanceof' (to check whether its a standard enum).

@lburgazzoli wdyt ?

E.g. StandardCode.ILLEGAL_OPTION could translate then to 'std.illegal.option' (or 'def' as prefix ? or no prefix ?)

The drawback would be, that with JSON serialization (as we use), still the name would be used. Also its a bit harder to map.

However, if we can live with '_' instead of '.' then we could make all enums lowercase too (e.g. std_illegal_option)